README.md

```diff
@@ -1,2 +1,92 @@
-# ibm-ingress-nginx-operator
-operator for ingress nginx
+# IBM Ingress nginx Operator
+
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+
+The IBM Ingress nginx Repo operator provides a Helm chart repository for storing and supplying IBM and local charts.
+
+For more information about the available IBM Cloud Platform Common Services, see the [IBM Knowledge Center](http://ibm.biz/cpcsdocs).
+
+## Supported platforms
+
+Red Hat OpenShift Container Platform 4.3 or newer installed on one of the following platforms:
+
+- Linux x86_64
+- Linux on Power (ppc64le)
+- Linux on IBM Z and LinuxONE
+  
+## Operator versions
+
+- 3.5.0
+- 3.6.0
+
+## Prerequisites
+
+Before you install this operator, you need to first install the operator dependencies and prerequisites:
+
+- For the list of operator dependencies, see the IBM Knowledge Center [Common Services dependencies documentation](http://ibm.biz/cpcs_opdependencies).
+
+- For the list of prerequisites for installing the operator, see the IBM Knowledge Center [Preparing to install services documentation](http://ibm.biz/cpcs_opinstprereq).
+
+## Documentation
+
+To install the operator with the IBM Common Services Operator follow the the installation and configuration instructions within the IBM Knowledge Center.
+
+- If you are using the operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak. For a list of IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+- If you are using the operator with an IBM Containerized Software, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcs_opinstall).
+
+## SecurityContextConstraints Requirements
+
+The Ingress nginx service requires running with the OpenShift Container Platform 4.3 default privileged Security Context Constraints (SCCs).
```

Comment:
Minor edit (capitalize Nginx)

The Ingress Nginx service requires running with the OpenShift Container Platform 4.3 default privileged Security Context Constraints (SCCs).

Reply:
Updated.